.github/workflows/test.yml

```diff
@@ -0,0 +1,77 @@
+name: Tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'develop'
+      - 'main'
+      - 'release/**'
+      - 'feature/ci'
+  pull_request:
+    branches:
+      - 'develop'
+      - 'main'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
I'm not convinced it's really all that useful to test four different Python versions; you could just test the version that runs on the production server.

Reply:
Four versions may indeed be overkill, but I'll keep 3.12 to facilitate installation at other places.